src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts

```diff
@@ -119,6 +120,29 @@ export class DotEditContentComponent implements OnInit, OnDestroy {
         }
     }
 
+    canDeactivate(): Observable<boolean> {
```

Comment:
Needs docs... not sure about the name (or the implementation) I think we need to discuss this with the whole team.